LanguageFeatures/Augmentations/augmenting_constructors_A17_t01.dart

```diff
@@ -2,21 +2,17 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion A redirecting generative constructor marked `augment` adds its
-/// redirection to the augmented constructor.
-///
-/// This converts it into a redirecting generative constructor, removing the
-/// potentially non-redirecting property of the constructor.
+/// @assertion An incomplete constructor can be completed by adding an
+/// initializer list and/or a body, or by adding a redirection.
 ///
 /// @description Checks that an augmenting redirecting generative constructor
 /// may add redirecting initializer to the augmented constructors initializer
```

Comment:
I think we should just call it a redirection:
```suggestion
/// may add a redirection to the augmented constructors initializer
```
Otherwise the terminology seems to imply that the redirection is just one element and there can be other initializer list elements. The fact that they both use `:` as a separator doesn't turn the redirection into an initializer list.